buildpacks/nodejs/lib/build.sh

```diff
@@ -106,6 +118,22 @@ install_or_reuse_node() {
 	fi
 }
 
+clear_cache_on_node_version_change() {
+	local layers_dir=$1
+	local prev_node_version
+	local curr_node_version
+
+	curr_node_version="$(echo $(node -v))"
```

Comment:
This is not passing shellcheck because of the "redundant echo" but I get a "node cmd not recognized" error without it. Do you have any suggestions to get around this?